core/config.h

```diff
@@ -276,6 +276,13 @@
 #define BH_ENABLE_GC_VERIFY 0
 #endif
 
+/* Enable buffered printf for builtin libc or not */
+#ifndef BH_BUILTIN_LIBC_BUFFERED_PRINTF
+#define BH_BUILTIN_LIBC_BUFFERED_PRINTF 0
+#define BH_BUILTIN_LIBC_BUFFERED_SIZE 128
+#define BH_BUILTIN_LIBC_BUFFERED_PREFIX
+#endif
```

Comment:
There are too many macros in config.h, these macro are only related to libc_builtin_wrapper.c, how about moving them to it?

Reply:
OK